components/Footer.js

```diff
@@ -42,11 +42,11 @@ export default function Footer() {
                   </div>
                 </div>
                 <ul className="justify-center">
-                  {/* <li className="py-2">
```

Comment:
should we keep it?

Reply:
heeyy @derberg :grinning: , I think, we need it, as we will be having `press` page with just links with a small summary (rather than paragraphs) - similar to [postman](https://www.postman.com/company/press-media/) as in https://github.com/asyncapi/website/pull/749#issuecomment-1175795440

![image](https://user-images.githubusercontent.com/69810789/181865147-f0353a95-6bd8-440d-bd0f-3ec047eb6162.png)